booster/tools/booster/Proxy.hs

```diff
@@ -376,7 +378,7 @@ respondEither cfg@ProxyConfig{statsVar, boosterState} booster kore req = case re
                                                 [ rpcLogs
                                                 , boosterResult.logs
                                                 , boosterStateSimplificationLogs
```

Comment:
Should we also filter these two? Or is the goal just to remove the `kore` simplification logs?
`boosterStateSimplificationLogs` should be empty because `simplifyExecuteState` disables those logs, but `boosterResult.logs` will still contain the ones that were produced during the execute request.

Reply:
Right, `boosterResult.logs` will actually contain simplification logs, so we should modify the Booster to no emit them in the first place. Let's indeed filter them out here for now...